projects/vdk-plugins/vdk-structlog/tests/test_structlog.py

```diff
@@ -113,6 +113,34 @@ def test_stock_fields_removal(log_format):
                 assert re.search(stock_field_reps[shown_field], test_log) is not None
 
 
+@pytest.mark.parametrize("log_format", ["console"])
+def test_custom_console_format(log_format):
+    custom_format_string = "%(asctime)s %(name)-12s %(levelname)-8s %(message)s"
+
+    with mock.patch.dict(
+        os.environ,
+        {
+            "VDK_LOGGING_METADATA": "timestamp,level,file_name,line_number,vdk_job_name",
+            "VDK_LOGGING_FORMAT": log_format,
+            "STRUCTLOG_CONSOLE_LOG_PATTERN": custom_format_string,
```

Comment:
I'm not sure this is the correct env variable. Shouldn't it be `VDK_LOGGING_CUSTOM_FORMAT`?

Reply:
Also, can you please add a description of the behavior to README.md. It should say that VDK_LOGGING_CUSTOM_FORMAT overrides VDK_LOGGING_METADATA and VDK_LOGGING_FORMAT.

Reply:
I don't like the way this test is set up. It can pass because of the additional configuration even if something is not right. It's better to have two tests: one that tests if the custom format works and another that tests if it overrides the other settings correctly.

Reply:
@DeltaMichael about the naming, please check the previous comments of Antoni I got it from there.
I will change the README but firstly will go over the tests

Reply:
@duyguHsnHsn My issue is not with the naming, you're setting a variable that doesn't exist. Please check again.

This

```python
"STRUCTLOG_CONSOLE_LOG_PATTERN": custom_format_string,
```

should be this

```python
"VDK_LOGGING_CUSTOM_FORMAT": custom_format_string,
```

because of this

```
STRUCTLOG_CONSOLE_LOG_PATTERN = "logging_custom_format"
```

`STRUCTLOG_CONSOLE_LOG_PATTERN` is just a const name in this case. You should be passing the value behind it, prefixed with `VDK`.

Reply:
Oh, you are right, sorry for the confusion